benchmark/buffers/buffer-from.js

```diff
@@ -8,14 +8,13 @@ const bench = common.createBenchmark(main, {
     'arraybuffer',
     'arraybuffer-middle',
     'buffer',
-    'uint8array',
     'string',
     'string-utf8',
     'string-base64',
     'object',
   ],
-  len: [10, 2048],
-  n: [2048]
+  len: [100, 2048],
+  n: [8e5]
```

Comment:
Buffer length of `10` should not matter performance wise in any application. They are so fast that we'd have to run millions of them to have a super tiny influence. Even a length of `100` seems very low.

A `buffer` is actually a `uint8array` with extra properties, thus I removed the redundant entry.

The iteration count was reduced to a reasonable number.